ci/before_deploy.sh

```diff
@@ -54,9 +54,17 @@ elif [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$ARCH" == "i386" ]; then
     # Make sure all files can be uploaded without permission errors
     sudo chown -R $USER:$USER "./target"
 elif [ "$TRAVIS_OS_NAME" == "windows" ]; then
-    choco install 7zip
+    choco install 7zip nuget.commandline
+    nuget install WiX
+
+    # Create zip archive
     7z a -tzip "./target/deploy/${name}-windows.zip" "./target/release/alacritty.exe" \
         "./target/release/winpty-agent.exe"
+
+    # Create msi installer
+    "./WiX.3.11.1/tools/candle.exe" -nologo -arch "x64" -ext WixUIExtension -ext WixUtilExtension -out "wix/alacritty.wixobj" "wix/alacritty.wxs"
```

Comment:
This will break and I'm not a fan of that at all. We shouldn't rely on a specific version of WiX.

The great thing is that we're running bash here though, so we are able to make use of the wonders of unix by changing this to `./WiX.*/tools/candle.exe`.

If it's possible to make sure that the version is not part of the wix folder name, that would of course be even better and less error-prone, but I'm not sure if it is. But maybe there's a nuget flag for this?

Reply:
The more commonly used MSI distributed WiX package set an environment variable with the path of the WiX installation however unfortunately chocolatey has a known bug preventing installing this package so I had to use the Nuget distribution as a workaround. Using a wildcard is a good suggestion though.